client/src/route/auth/ConfigItems.tsx

```diff
@@ -0,0 +1,85 @@
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline';
+import { Tooltip } from '@mui/material';
+import * as React from 'react';
+import { validationType } from './VerifyConfig';
+
+const ConfigIcon = (toolTipTitle: string, icon: JSX.Element): JSX.Element => (
+  <Tooltip
+    title={toolTipTitle}
+    PopperProps={{ container: document.getElementById('root') }}
+  >
+    {icon}
+  </Tooltip>
+);
+
+export const getConfigIcon = (
+  validation: validationType,
+  label: string,
+): JSX.Element => {
+  let icon = <ErrorOutlineIcon color="error" data-testid="error-icon" />;
+  let toolTipTitle = `${label} threw the following error: ${validation.error}`;
+  const configHasStatus = validation.status !== undefined;
+  const configHasError = validation.error !== undefined;
+  if (!configHasError) {
+    const statusMessage = configHasStatus
+      ? `${validation.value} responded with status code ${validation.status}.`
+      : '';
+    const validationStatusIsOK =
+      configHasStatus &&
+      ((validation.status! >= 200 && validation.status! <= 299) ||
```

Comment:
please use [htp-status-codes](https://www.npmjs.com/package/http-status-codes) package to check the return codes.